clang/test/SemaSYCL/array-kernel-param.cpp

```diff
@@ -114,90 +120,19 @@ int main() {
 // CHECK-NEXT: MemberExpr {{.*}}__init
 
 // Check kernel_D parameters
-// CHECK: FunctionDecl {{.*}}kernel_D{{.*}} 'void (int, int, int, int, int, int, int, int, int, int, int, int, int, int, int, int, int, int, int, int)'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_a 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_x 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_y 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_x 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_y 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_c 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_a 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_x 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_y 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_x 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_y 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_inner_z 'int'
-// CHECK-NEXT: ParmVarDecl {{.*}} used _arg_foo_c 'int'
+// CHECK: FunctionDecl {{.*}}kernel_D{{.*}} 'void (int, int, int)'
```

Comment:
This is incorrect. You should not remove the existing test. You need to rename one of the kernels and keep both tests.

Reply:
I did not intend to remove a test. Let me check.